docs/running-rules.md

```diff
@@ -31,7 +31,9 @@ $ semgrep -e '$X == $X' --lang=py path/to/src
 $ semgrep --config path/to/yaml
 ```
 
-# Managing findings
+# Ignoring findings
```

Comment:
n.b. - we'll need to add redirects for the docs before merging title changes (because they are used in existing links)

Reply:
Sadly we cannot use redirects here, because any content after the # is not passed to the server. Thoughts on how to handle? IMO this is an argument for having more pages rather than fewer pages with lots of local anchor # links.

Reply:
A crude way would just be to add an HTML element to catch that link and place it right next to the updated header. Something like: `<div id="managing-findings"></div>`

Reply:
Good point @pabloest . I ended up just putting the header back in but linking ignore findings.